packages/flutter/lib/src/painting/text_span.dart

```diff
@@ -196,78 +202,92 @@ class TextSpan extends DiagnosticableTree {
 
   /// Walks this text span and its descendants in pre-order and calls [visitor]
   /// for each span that has text.
-  bool visitTextSpan(bool visitor(TextSpan span)) {
+  @override
+  bool visitChildren(InlineSpanVisitor visitor) {
     if (text != null) {
       if (!visitor(this))
         return false;
     }
     if (children != null) {
-      for (TextSpan child in children) {
-        if (!child.visitTextSpan(visitor))
+      for (InlineSpan child in children) {
+        if (!child.visitChildren(visitor))
           return false;
       }
     }
     return true;
   }
 
   /// Returns the text span that contains the given position in the text.
-  TextSpan getSpanForPosition(TextPosition position) {
-    assert(debugAssertIsValid());
+  @override
```

Comment:
Remove the doc comment above? Seems like the base class has a more informative doc comment that currently gets overwritten by this comment.

Reply:
(This may also be true for some of the other overwrites that still have their own doc comment. Unless the override has a more specific doc comment than the base it should properly be removed.)

Reply:
Yeah, looks like there a re a couple of duplicated doc comments between base class and subclass. Those will probably go out of sync quickly. If there is no difference between the comment on the base and subclass, the comment on the subclass should be removed.